tests/integration/test_main.py

```diff
@@ -76,3 +84,38 @@ def test_integrate(juju: jubilant.Juju):
     data = json_output["interface-consumer/0"]["relation-info"][0]["application-data"]
     assert(data.get("db_nb_connection_str"))
     assert(data.get("db_sb_connection_str"))
+
+def test_certificates_integration(juju: jubilant.Juju):
+    juju.deploy(microovn_charm_path)
+    juju.add_unit("microovn")
+    juju.deploy(token_distributor_charm_path)
+    juju.deploy("self-signed-certificates")
+    juju.integrate("microovn","microcluster-token-distributor")
+    juju.integrate("microovn","self-signed-certificates")
+    juju.wait(jubilant.all_active)
+    juju.wait(lambda _: "CA certificate updated, new certificates issued" in juju.debug_log())
+    destination = juju.status().apps["microovn"].units["microovn/1"].public_address
+    destination = destination + ":6643"
+    command_str = "openssl s_client -connect {0} -verify_return_error".format(destination)
```

Comment:
nit: the `-verify_return_error` is of no use, as the command returns error regardless.